docs/batch-api.md

```diff
@@ -20,7 +20,12 @@ await op1.confirmation();
 await op2.confirmation();
 
 /*
- * Error Message returned by the node:
+ * Error Message returned by the node (since Kathmandu):
+ * Error while applying operation opHash: 
+ * Only one manager operation per manager per block allowed (found opHash2 with Xtez fee. 
+ * You may want to use --replace to provide adequate fee and replace it).
```

Comment:
Minor, ignore it if too much trouble:

"to provide adequate fee"  -> "to provide an adequate fee"